Objects/dictobject.c

```diff
@@ -4285,3 +4479,970 @@ _PyDictKeys_DecRef(PyDictKeysObject *keys)
 {
     DK_DECREF(keys);
 }
+
```

Comment:
But new OrderedDict uses some internal static functions and variables.
I don't want to make it `extern` functions.
Do you prefer `#include "odictobject.h"`?